test/exercism/use_cases/attempt_test.rb

```diff
@@ -190,8 +196,10 @@ def test_attempt_sets_exercise_as_current
 
   def test_attempt_sets_completed_exercises_as_current
     refute user.working_on?(Problem.new('ruby', 'one'))
-    attempt = Attempt.new(user, "CODE", 'ruby/one/one.rb').save
-    assert user.working_on?(Problem.new('ruby', 'one'))
+    solution = { "ruby/one/one.rb" => "CODE" }
+    opts = { track: "ruby", slug: "one" }
+    Attempt.new(user, Iteration.new(solution, opts)).save
+    assert_equal true, user.working_on?(Problem.new('ruby', 'one'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.